src/content/docs/en/contribute.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Contribute to the Docs
+description: A guide to contributing to Astro Docs.
+i18nReady: true
+---
+import Button from '~/components/Button.astro'
+import ContributorList from '~/components/ContributorList.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+We welcome contributions of any size and contributors of any skill level. As an open source project, we believe in giving back to our contributors. We are happy to help with guidance on PRs, technical writing, and turning any feature idea into a reality.
+
+## Ways to Contribute
+
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions and participating in our "Pinned" issue maintenance tasks! 
+
+Every PR, especially translation PRs, needs reviewers! Reviewing PRs and leaving comments, suggestions or an approving "LGTM!" ("Looks Good To Me!") is a great way to get started on Team Docs, and to learn more about Astro.
+
+## Contributor manuals
```

Comment:
Maybe this is just me, but I think it's strange to address "Contributor manuals" & "Community manuals" and then only make a list of "guides".

It's just wording, and you're the OG of Astro docs, so feel free to ignore 😅

Reply:
I agree with Elian here - I prefer "Contributor Guides"

Reply:
Changing it!